applications/ShapeOptimizationApplication/python_scripts/response_function_factory.py

```diff
@@ -64,25 +65,10 @@ def __checkIfGivenResponseFunctionIsAlreadyDefined( self, response_id ):
 
     # --------------------------------------------------------------------------
     def __createAndAddGivenResponse( self, response_id, response_settings ):
```

Comment:
could be adjusted to new style guide for python. I can do that in a small commit.